apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -179,6 +191,26 @@ const MeetingForm: React.FC = () => {
         value={blurValue}
         label="Blur"
       />
+      {/* <Video uplink and downlink policies/> */}
```

Comment:
Can we remove angle brackets for <Video uplink and downlink policies/>? Looks like a component.

Reply:
I just follow the BlurSelection stuff below. Thought it was weird but hey consistent eh :D